.babelrc

```diff
@@ -0,0 +1,21 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions", "ie >=  10"]
```

Comment:
We target ie11 now :)

Reply:
Also, "last 2 versions" is dangerous. Do you really want to target the last 2 versions of Opera ?